internal/resolver/dns/dns_resolver_test.go

```diff
@@ -1439,7 +1440,7 @@ func TestCustomAuthority(t *testing.T) {
 
 		b := NewBuilder()
 		cc := &testClientConn{target: "foo.bar.com", errChan: make(chan error, 1)}
-		r, err := b.Build(resolver.Target{Endpoint: "foo.bar.com", Authority: a.authority}, cc, resolver.BuildOptions{})
+		r, err := b.Build(resolver.Target{Endpoint: "foo.bar.com", Authority: a.authority, URL: url.URL{Host: a.authority}}, cc, resolver.BuildOptions{})
```

Comment:
Could you please rewrite this as follows to make it a little bit more readable:

```golang
  target := resolver.Target{
    Endpoint: "foo.bar.com", 
    Authority: a.authority, 
    URL: url.URL{Host: a.authority},
  }
  r, err := b.Build(target, cc, resolver.BuildOptions{})
```

Reply:
done